x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -43,15 +46,26 @@ export const ComplianceDashboard = () => {
       <div
         data-test-subj={DASHBOARD_CONTAINER}
         css={css`
-          max-width: 1600px;
+          max-width: 1440px;
```

Comment:
can you clarify why do we need fixed size? i know the value just changed, but still - can we avoid it?

Reply:
what do you mean? we dont want to have the dashboard smeared over the widescreen and such, it uses max width to stay compact and centered to the screen. pavel used 1440px and that's why this is the set max width.